src/frequenz/sdk/microgrid/_power_distributing/_component_managers/_ev_charger_manager/_ev_charger_manager.py

```diff
@@ -66,6 +66,10 @@ def __init__(
         self._api_power_request_timeout = api_power_request_timeout
         self._fallback_power = fallback_power
 
+        assert (
+            self._fallback_power == Power.zero()
+        ), "Non-zero fallback power is not supported for EV chargers."
+
```

Comment:
Same, maybe you could mention this in the `fallback_power` docs 4 lines above?
It only says:
```
            fallback_power: The power to assume a battery has if the battery is not
                reachable.
```

Reply:
Updated.